Sources/Kitura/Kitura.swift

```diff
@@ -22,14 +22,31 @@ import Dispatch
 
 // MARK Kitura
 
-/// A set of helper functions to make it easier to create, start, and stop Kitura based servers.
+/**
+ A set of helper functions to make it easier to create, start, and stop Kitura based servers.
+ ### Usage Example: ###
+ In this example, we create a function called run, inside a server `Application.swift` file, which will create a Kitura server on the specified port, using the given router and then run the created server. This function would then be the last call in your `main.swift` file.
+ ```swift
+ let router = Router()
+ let port: Int = 8080
+ public func run() throws{
+ ...
+ Kitura.addHTTPServer(onPort: port, with: router)
+ Kitura.run()
+ }
+ ```
+ */
 public class Kitura {
 
     /// Add an HTTPServer on a port with a delegate.
     ///
     /// The server is only registered with the framework, it does not start listening
     /// on the port until Kitura.run() or Kitura.start() is called.
     ///
+    ///### Usage Example: ###
+    ///```swift
+    /// Kitura.addHTTPServer(onPort: port, with: router)
+    ///```
```

Comment:
The last bit of the keepAlive parameter description needs fixing to say "Keep-Alive will not be permitted" rather than "Keep-Alive will be not be permitted".